test/unit/rules/no-multiple-empty-lines-test.js

```diff
@@ -26,6 +26,7 @@ generateRuleTests({
   bad: [
     {
       template: '<div>foo</div>\n\n\n<div>bar</div>',
+      fixedTemplate: '<div>foo</div>\n<div>bar</div>',
```

Comment:
I'm not sure this is right. Shouldn't we collapse collapse any number of blank lines to just one? This collapses it to no blank lines.

```
foo


bar
```

should be fixed to:

```
foo

bar
```

which is equivalent to having `\n\n`.

Reply:
oh yeah, done!